cmd/cluster/core/dump.go

```diff
@@ -105,6 +105,8 @@ func DumpCluster(ctx context.Context, opts *DumpOptions) error {
 	}
 	cmd.WithNamespace(opts.Namespace).Run(ctx, objectNames...)
 
+	cmd.Run(ctx, objectType(&corev1.Node{}))
```

Comment:
Why make this its own `cmd.Run` rather than including it in the `resources` list below?

Reply:
because those are namespaced and are output for different namespaces in separate invocations?